graylog2-web-interface/webpack.config.js

```diff
@@ -102,6 +103,15 @@ const webpackConfig = merge.smart(coreConfig, {
       },
       chunksSortMode: core.sortChunks,
     }),
+    // Create SBOM files for graylog-server frontend dependencies.
+    new CycloneDxWebpackPlugin({
+      specVersion: '1.5',
+      rootComponentAutodetect: false,
+      rootComponentType: 'application',
+      rootComponentName: 'graylog-server',
+      outputLocation: '../cyclonedx',
+      includeWellknown: false,
+    }),
```

Comment:
Although this does not seem to run during the development build, we could still move this to the production section in `webpack/core.js`, just to clarify that this is only relevant for the production build.

Reply:
@linuspahl I think it might be useful in development as well. Like getting a list of all dependencies and transitive dependencies.

Reply:
I had the impression that it is not creating any files when you just run `yarn start`, which is executing the development path, only when you run `yarn build`.

Reply:
@linuspahl Oh, you are right. I didn't realize the plugin doesn't run when executing `yarn start`. I moved the plugin to `webpack/core.js`. Thanks!